lib/core/utils/get-friendly-uri-end.js

```diff
@@ -19,6 +19,10 @@ function splitString (str, splitIndex) {
 	return [str.substring(0, splitIndex), str.substring(splitIndex)];
 }
 
+function trimRight (str) {
```

Comment:
Use prototype methods - `.trimRight()` or `.trimEnd()`?
Need IE Support?

Reply:
yeah, needs to work in IE.